devices/Adxl343/ADXL343/ADXL343.cs

```diff
@@ -0,0 +1,1748 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using nanoFramework.Hardware.Esp32;
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Numerics;
+using System.Threading;
+
+namespace ADXL343
+{
+    public class ADXL343
+    {
+        private const int resolution = 1024;
```

Comment:
Note that once you'll add the linter, you'll get quite a lot of feedbacks on the naming conventions, please address them, you'll have few other things pointed out, so I won't do it now. I'll point few more elements that the linter may not catch